ebpf/c/tracer-ebpf.c

```diff
@@ -451,19 +451,16 @@ static void update_conn_stats(
     t.sport = ntohs(t.sport); // Making ports human-readable
     t.dport = ntohs(t.dport);
 
+    // initialize-if-no-exist the connection stat, and load it
+    conn_stats_ts_t empty = {};
+    bpf_map_update_elem(&conn_stats, &t, &empty, BPF_NOEXIST);
     val = bpf_map_lookup_elem(&conn_stats, &t);
+
     // If already in our map, increment size in-place
     if (val != NULL) {
```

Comment:
technically the val != NULL check is always going to be true, but the BPF verifier will reject this code if we don't check